Source/MonoGame.Extended.Tests/UnsafeCodeTests.cs

```diff
@@ -0,0 +1,72 @@
+﻿using System;
+using System.Text;
+using NUnit.Framework;
+
+namespace MonoGame.Extended.Tests
+{
+    [TestFixture]
+    public class UnsafeCodeTests
+    {
+        [Test]
+        public void Unsafe_Copy_Test()
+        {
+            var a = new byte[100];
+            var b = new byte[100];
+            for (var i = 0; i < 100; ++i)
+            {
+                a[i] = (byte)i;
+            }
+            UnsafeCopy(a, 0, b, 0, 100);
+            var stringBuilder = new StringBuilder();
+            stringBuilder.Append("The first 10 elements are: ");
+            for (var i = 0; i < 10; ++i)
+            {
+                stringBuilder.Append(b[i] + " ");
+            }
+
+            Assert.AreEqual(stringBuilder.ToString(), "The first 10 elements are: 0 1 2 3 4 5 6 7 8 9 ");
+        }
+
```

Comment:
If we don't need it, we don't need it. When / if we do need it we'll add it then.